.chloggen/add-jmx-metrics-gatherer-1.20.1-alpha.yaml

```
@@ -0,0 +1,4 @@
+change_type: enhancement
+component: jmxreceiver
+note: Add the JMX metrics gatherer version 1.20.1-alpha to the supported jars hash list
+issues: []
```

I'll see about getting the PR number in here automatically in the next release

```suggestion
issues: [ 16437 ]
```